### PR TITLE
Gh 1223/show contracts v1.3.0 as "up to date" in safe settings

### PR DESCRIPTION
**Pull request by MouazAlzahabi**
Handles #1223

### CODE REVIEW

File: Multisig/Data/Services/Utils/String+Utils.swift [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+//
+//  Created by Moaaz on 7/19/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+    func compareNumeric(_ value: String) -> ComparisonResult {
+        return compare(value, options: .numeric)
```
Comment (DmitryBespalov) [inline Multisig/Data/Services/Utils/String+Utils.swift]: I think we already have imported the "Version" swift package, I think this should be used in order to compare versions.

File: MultisigTests/Logic/Models/GnosisSafeTests.swift [hunk 2]
```diff
@@ -10,57 +10,18 @@ import XCTest
...
-            "0xAC6072986E985aaBE7804695EC2d8970Cf7541A2",
-            "0x8942595A2dC5181Df0465AF0D7be08c8f23C93af",
-        ]
-
-        let unknown: [Address] = [
-            "0x3b1c2b0940C85458197E0D18690805d6F89547eE",
-            "0x976DC99c50B916Ea9b5275979059BCe9f1A0B1D1",
-            "0xD5D4763AE65aFfFD82e3aEe3ec9f21171A1d6e0e",
-        ]
+        let supported: [String] = ["1.0.0", "1.1.0", "1.2.3"]
```
Comment (DmitryBespalov) [inline MultisigTests/Logic/Models/GnosisSafeTests.swift]: so the supported is just between 1.0.0 and 1.3.0?

In that case, tests should test the edge cases, inside the interval, and outside the interval.

- 1.0.0
- 1.3.0
- 1.3.1
- 1.4.0
- 0.1.0
- 1.1.1

Comment (DmitryBespalov) [reply MultisigTests/Logic/Models/GnosisSafeTests.swift]: Well, actually, I would test with explicit specification the minimum and maximum version, so that the test won't break when we add/change the boundaries.

**Review by DmitryBespalov — APPROVED** [review]
